### PR TITLE
fix: circleci config.yml issue

### PR DESCRIPTION
**Pull request by thapasya-m**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -25,7 +25,7 @@ jobs:
           key: v1-deps-client-cache-{{ checksum "./client/yarn.lock" }}
       - run:
           name: Test client
-          command: yarn test:client
+          command: cd client && yarn && yarn test
```
Comment (nimish-gupta) [inline .circleci/config.yml]: why are you doing the cd client when you have already set the working_directory?

File: .circleci/config.yml [hunk 2]
```diff
@@ -51,7 +51,7 @@ jobs:
           key: v1-deps-server-cache-{{ checksum "./server/yarn.lock" }}
       - run:
           name: Test server
-          command: yarn test:server
+          command: cd server && yarn && yarn test
```
Comment (nimish-gupta) [inline .circleci/config.yml]: why are you doing the cd server when you have already set the working_directory?

Comment (thapasya-m) [reply .circleci/config.yml]: > why are you doing the cd server when you have already set the working_directory?

Similar issue with test server job. Attaching build failed error

File: .circleci/config.yml (hunk 1 above)
Comment (thapasya-m) [reply .circleci/config.yml]: > why are you doing the cd client when you have already set the working_directory?
when i do not add the cd client i recieve the following build error

**Pull request merged**